.github/workflows/ci.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
                     key: ${{ runner.os }}-doctor-rst-${{ steps.extract_base_branch.outputs.branch }}
 
             -   name: "Run DOCtor-RST"
-                uses: docker://oskarstark/doctor-rst:1.45.0
+                uses: docker://oskarstark/doctor-rst:1.47.0
```

Comment:
We can also enable rule NoDuplicateUseStatements

Reply:
This one is failing, I created a PR to check if the regression is in 1.47 or 1.46.1 

See #18326

Reply:
I enabled the rule

Reply:
```suggestion
                uses: docker://oskarstark/doctor-rst:1.47.1
```